.gitignore

```diff
@@ -55,3 +55,4 @@ tables/iRO/Restart/skillnametable-custom.txt
 tables/**/portalsLOS.txt
 tables/**/npcs.txt
 tables/**/monsters.txt
+*.xpm
```

Comment:
What's this?

Reply:
My fault, but this is a xpm file auto generated by map.pl
Will be veru usefull